third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run-expected.txt

```diff
@@ -162,42 +162,42 @@ Auditing: Registers a service worker that controls page and `start_url`
 Auditing: Has a `<meta name="viewport">` tag with `width` or `initial-scale`
 Computing artifact: ViewportMeta
 Auditing: First Contentful Paint
-Computing artifact: FirstContentfulPaint
+Computing artifact: FirstContentfulPaint$1
 Computing artifact: LanternFirstContentfulPaint
 Computing artifact: PageDependencyGraph
 Computing artifact: LoadSimulator
 Computing artifact: NetworkAnalysis
 Auditing: Largest Contentful Paint
-Computing artifact: LargestContentfulPaint
+Computing artifact: LargestContentfulPaint$1
 Computing artifact: LanternLargestContentfulPaint
 Auditing: First Meaningful Paint
-Computing artifact: FirstMeaningfulPaint
+Computing artifact: FirstMeaningfulPaint$1
 Computing artifact: LanternFirstMeaningfulPaint
 Auditing: Speed Index
-Computing artifact: SpeedIndex
+Computing artifact: SpeedIndex$1
 Computing artifact: LanternSpeedIndex
 Computing artifact: Speedline
 Auditing: Screenshot Thumbnails
 Auditing: Final Screenshot
 Computing artifact: Screenshots
 Auditing: Total Blocking Time
-Computing artifact: TotalBlockingTime
+Computing artifact: TotalBlockingTime$1
 Computing artifact: LanternTotalBlockingTime
 Computing artifact: LanternInteractive
 Auditing: Max Potential First Input Delay
-Computing artifact: MaxPotentialFID
+Computing artifact: MaxPotentialFID$1
 Computing artifact: LanternMaxPotentialFID
 Auditing: Cumulative Layout Shift
-Computing artifact: CumulativeLayoutShift
+Computing artifact: CumulativeLayoutShift$1
```

Comment:
was this fixed already?

Reply:
for _computed artifacts_ the exact name doesn't matter, so I didn't bother. would have to add a getter `name()` to ComputedArtifact class, which only impacts the log message  https://github.com/GoogleChrome/lighthouse/blob/f90efbf/lighthouse-core/computed/computed-artifact.js#L30